XCUITest/BaseTestCase.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  BaseTestCase.swift
+//  Blockzilla
+//
+//  Created by mozilla on 11/9/16.
+//  Copyright © 2016 Mozilla. All rights reserved.
+//
```

Comment:
Please replace this with the MPL header used in other Swift files (here and other files).